docs/analyzer/user_guide.md

```diff
@@ -1378,6 +1379,54 @@ then the results of the analysis can be printed with
 CodeChecker parse ./my_plists
 ```
 
+## `fixit` <a name="fixit"></a>
+
+ClangTidy is able to provide suggestions on automatic fixes of reported issues.
+For example there is a ClangTidy checker which suggests using
+`collection.empty()` instead of `collection.size() != 0` expression. These
+simple changes can be applied directy in the source code. `CodeChecker fixit`
+command handles these automatic fixes.
+
+```
+usage: CodeChecker fixit [-h] [-l]
+                         [--checker-name [CHECKER_NAME [CHECKER_NAME...]]]
+                         [--file [FILE [FILE ...]]]
+                         [--verbose {info,debug,debug_analyzer}]
+                         folder [folder ...]
+
+Some analyzers may suggest some automatic bugfixes. Most of the times these are
+style issues which can be fixed easily. This command handles the listing and
+application of these automatic fixes.
+
+Besides the provided filter options you can pipe the JSON format output of
+"CodeChecker cmd diff" command to filter automatic fixes only on new reports:
+CodeChecker cmd diff -b dir1 -n dir2 -o json --new | CodeChecker fixit dir2
```

Comment:
Most of the time users are comparing a remote run and a directory so it would be better to give one more example or rewrite this one where you use a remote run instead of a directory at -b option:
```
CodeChecker cmd diff -b remote_run -n dir -o json --new | CodeChecker fixit dir
```

Also, I think we should redirect the standard error to /dev/null.
```sh
CodeChecker cmd diff -b remote_run -n dir -o json --new 2>/dev/null | CodeChecker fixit dir
```

This way if I forgot to give a directory to the fixit command it will not throw the `_io.TextIOWrapper` exception mentioned above.

Reply:
Redirecting standard error is just a symptomatic treatment, since exception is still thrown but goes to `/dev/null`.